src/polyfem/io/OutData.cpp

```diff
@@ -94,7 +94,28 @@ namespace polyfem::io
 
 				for (int n = 0; n < normals.rows(); ++n)
 				{
-					normals.row(n) = normals.row(n) * vals.jac_it[n];
+					trafo = vals.jac_it[n].inverse();
```

Comment:
was this not already fixed?

Reply:
This was already fixed for the simulation, but I guess not fixed for the output.